### DISCUSSION TITLE
Database configuration for statistics

### DISCUSSION THREAD

**Discussion by francescodiperna** [OxalisCommunity/oxalis#554] [General]
Hi, 
reading the documentation I realized that is possible to configure oxalis for statistics of document exchanged.
I tried this configurazion:
```

# Database
oxalis.database.datasource = dbcp
oxalis.database.dbcp.validation = select 1
oxalis.database.driver.class = com.mysql.jdbc.Driver
oxalis.database.driver.path = "file:///oxalis/ext/mysql-connector-java-8.0.27.jar"
oxalis.database.jdbc.connection = "jdbc:mysql://HOSTNAME:3306/oxalis"
oxalis.database.jdbc.username = root
oxalis.database.jdbc.password = xxxx
```

but nothing is written [in the table raw_stats](https://github.com/OxalisCommunity/oxalis/blob/master/oxalis-legacy/oxalis-statistics/src/main/resources/sql/raw_stats-mysql.sql).
Anybody can help me?
Kind regards

Di Perna Francesco

**Comment by francescodiperna** [comment]
Anybody can help me?

**Reply by soehtaike92** [reply]
Just curious if you managed to get it working in the end? I've upgraded to latest version 6 and still haven't managed to get any data written to raw_stats table in MySQL yet. thx. There's no error in the oxalis log file either.

**Reply by francescodiperna** [reply]
Hi, I didn't.  I will retry in the next weeks.

**Reply by DioMeo** [reply]
Hello Francesco, during the installation phase for the statistics we are having the same problem reported by You.
In the end, were you able to resolve the call to the MySQL database?
This is the one inserted in our oxalis.conf document:

oxalis.database.datasource = dbcp
oxalis.database.dbcp.validation = select 1
oxalis.database.driver.class = com.mysql.jdbc.Driver
oxalis.database.driver.path = "/usr/share/java/mysql-connector-j-8.0.31.jar"
oxalis.database.jdbc.connection = "jdbc:mysql://localhost:3306/peppol_stats_test"
oxalis.database.jdbc.username = root
oxalis.database.jdbc.password = xxxxxxxx

Thanks.

**Comment by francescodiperna** [comment]
> Hello Francesco, during the installation phase for the statistics we are having the same problem reported by You. In the end, were you able to resolve the call to the MySQL database? This is the one inserted in our oxalis.conf document:
> 
> oxalis.database.datasource = dbcp oxalis.database.dbcp.validation = select 1 oxalis.database.driver.class = com.mysql.jdbc.Driver oxalis.database.driver.path = "/usr/share/java/mysql-connector-j-8.0.31.jar" oxalis.database.jdbc.connection = "jdbc:mysql://localhost:3306/peppol_stats_test" oxalis.database.jdbc.username = root oxalis.database.jdbc.password = xxxxxxxx
> 
> Thanks.

Hi, I wasn't able to solve this, sorry.

**Comment by DioMeo** [comment]
Hello, Francesco.
No problem, we'll find a different way to solve it.
Thank for reply to me.

**Reply by francescodiperna** [reply]
Hi DioMeo, I think we are working in the same tasks, can I have your email? Maybe togher we can reach the goal :-)

Francesco

**Reply by DioMeo** [reply]
Hi Francesco, in the end we opted to recover the information directly from the log files, because doesn't work the connection to a db, as Aaron reply to my question here: https://github.com/OxalisCommunity/oxalis/issues/654

**Reply by francescodiperna** [reply]
Thanks

**Comment by francescodiperna** [comment]
Thanks
